docs/bundle_spec.md

```diff
@@ -0,0 +1,66 @@
+# Policy bundle specification
+
+This document describes the contract and API for policy bundles that run in the policy engine.
+
+## Policy bundle requirements
+
+1. Policy bundles must be a GZipped Tar file
+2. Policy bundles must contain one or more policies that meet the requirements defined in the
+   [Policies specification](policy_spec.md)
+3. Policies must be stored within a top-level `rules` directory
+
+## Optional bundle contents
+
+### `metadata.json`
+
+A `metadata.json` file in the root of the bundle will be interpreted as "bundle metadata". This
```

Comment:
```suggestion
A `manifest.json` file in the root of the bundle will be interpreted as "bundle metadata". This
```

There's a slight inconsistency as down below in the file tree there's `manifest.json`.  I actually like manifest more, since it prevents confusion -- if we talk about "metadata", it refers to the rule thing, if we talk about "manifest", it refers to the bundle thing.